spatio_temporal_voxel_layer/src/spatio_temporal_voxel_layer.cpp

```diff
@@ -599,11 +599,6 @@ void SpatioTemporalVoxelLayer::reset(void)
 
   current_ = false;
   was_reset_ = true;
-
-  observation_buffers_iter it = _observation_buffers.begin();
-  for (; it != _observation_buffers.end(); ++it) {
-    (*it)->ResetLastUpdatedTime();
```

Comment:
This should remain, no? The costmap layers for the moment still do this https://github.com/ros-navigation/navigation2/blob/main/nav2_costmap_2d/plugins/obstacle_layer.cpp#L758-L774

Reply:
Yeah, that's true.
This change was a bit prematurely following [this issue](https://github.com/ros-navigation/navigation2/issues/4734). But that needs to be it's own PR.
Changes are now reverted on the reset() function.